config/initializers/clowder_config.rb

```diff
@@ -36,8 +36,8 @@ def self.instance
         options["databaseUsername"] = config.database.username
         options["databasePassword"] = config.database.password
 
-        options["SOURCES_URL"] = exists("SOURCES_URL", options["endpoints"]["sources-api"])
-        options["CATALOG_INVENTORY_INTERNAL_URL"] = options["endpoints"]["catalog-inventory-api"]
+        options["SOURCES_URL"] = exists("SOURCES_URL", options["endpoints"]["sources-api-svc"])
+        options["CATALOG_INVENTORY_INTERNAL_URL"] = options["endpoints"]["catalog-inventory-api-v2"]
```

Comment:
@hsong-rh The suffix -v2 in the service name is really confusing. Why doesn't sources have a v2 in its name.

Reply:
There is a historical reason. It is just a name. Not much meaning.